README.md

```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
```

Comment:
The table you inserted on top seems nice. You also mentioned that the output is PDF. document. While learning from your code, I’m also quite curious about the difference between pdf output and other formats(?).